lib/index.js

```diff
@@ -10,10 +10,20 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-/* globals process, console, Q, JSON, escape, verifyToken, crypto */
+/* globals process, console, SDK_VERSION */
 /*jshint maxcomplexity:15 */
 
-var SDK_VERSION = '<%= sdkversion %>'; // Will be replaced by grunt
+require('./vendor/polyfills');
+
+var Q = require('q');
```

Comment:
One thing that I've started doing in my node files is setting up requires like this:
```javascript
var Q         = require('q'),
    sdkCrypto = require('./crypto'),
    etc
```
i.e. match up the = require('module') part